<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gallery</class>
 <widget class="QMainWindow" name="Gallery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gallery</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>731</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="Control">
     <attribute name="title">
      <string>Control</string>
     </attribute>
     <widget class="QPushButton" name="PushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Button 1</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Button2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>A Menu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>This Button</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Happens to Have</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QKeySequenceEdit" name="keySequenceEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Text frame</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>50</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tickmarks</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Word warp</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>100</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Top</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>100</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Bottom</string>
      </property>
     </widget>
     <widget class="QFontComboBox" name="fontComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>461</width>
        <height>271</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo cosnsequat. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo cosnsequat. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Apple</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Banana</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Orange</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tab Position</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Itemviews">
     <attribute name="title">
      <string>Itemviews</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1</width>
        <height>1</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>731</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Styles">
     <attribute name="title">
      <string>Styles</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>90</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>170</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="geometry">
       <rect>
        <x>69</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>140</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>230</y>
        <width>100</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>220</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Layouts">
     <attribute name="title">
      <string>Layouts</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
